transport/pom.xml

```diff
@@ -59,6 +59,16 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
```

Comment:
@antonkharenko in the last commit I have added log4j-test cfg in every module with test code. Dependency on log4j marked as ``optional``.